docs/content/2-widgets/2-containers/splits.md

```diff
@@ -37,6 +37,16 @@ core.NewText(sp).SetText("Fourth")
 sp.SetTiles(core.TileSpan, core.TileSecondLong)
 ```
 
+```Go
```

Comment:
You should add text here that says something like `There are various tile options for splits:`